stabilize.md

```diff
@@ -49,39 +49,137 @@ Flap Semantics
 
     syntax MCDStep ::= "Flap" "." FlapStep
  // --------------------------------------
+    rule <k> step [ Flap . FAS:FlapAuthStep ] => Flap . push ~> Flap . auth ~> Flap . FAS ~> Flap . catch ... </k>
+    rule <k> step [ Flap . FS               ] => Flap . push ~>                Flap . FS  ~> Flap . catch ... </k>
+      requires notBool isFlapAuthStep(FS)
 
     syntax FlapStep ::= FlapAuthStep
  // --------------------------------
 
     syntax FlapAuthStep ::= AuthStep
  // --------------------------------
+    rule <k> Flap . auth => . ... </k>
+         <msg-sender> MSGSENDER </msg-sender>
+         <flap-ward> ... MSGSENDER |-> true ... </flap-ward>
+
+    rule <k> Flap . auth => Flap . exception ... </k>
+         <msg-sender> MSGSENDER </msg-sender>
+         <flap-ward> ... MSGSENDER |-> false ... </flap-ward>
 
     syntax FlapAuthStep ::= WardStep
  // --------------------------------
+    rule <k> Flap . rely ADDR => . ... </k>
+         <flap-ward> ... ADDR |-> (_ => true) ... </flap-ward>
 
-    syntax FlapAuthStep ::= "init" Address Address
- // ----------------------------------------------
+    rule <k> Flap . deny ADDR => . ... </k>
+         <flap-ward> ... ADDR |-> (_ => false) ... </flap-ward>
 
     syntax FlapStep ::= StashStep
  // -----------------------------
+    rule <k> Flap . push => . ... </k>
+         <flapStack> (.List => ListItem(FLAP)) ... </flapStack>
+         <flap-state> FLAP </flap-state>
+
+    rule <k> Flap . pop => . ... </k>
+         <flapStack> (ListItem(FLAP) => .List) ... </flapStack>
+         <flap-state> _ => FLAP </flap-state>
+
+    rule <k> Flap . drop => . ... </k>
+         <flapStack> (ListItem(_) => .List) ... </flapStack>
 
     syntax FlapStep ::= ExceptionStep
  // ---------------------------------
+    rule <k>                      Flap . catch => Flap . drop ... </k>
+    rule <k> Flap . exception ~>  Flap . catch => Flap . pop  ... </k>
+    rule <k> Flap . exception ~> (Flap . FS    => .)          ... </k>
+      requires FS =/=K catch
+```
 
-    syntax FlapStep ::= "kick" Int Int
- // ----------------------------------
+- kick(uint lot, uint bid) returns (uint id)
+- Starts a new surplus auction for a lot amount
 
+**TODO** The `end` field of the Bid (TIME +Int 172800) is supposed to be two days from the current time.
+
+```k
+    syntax FlapAuthStep ::= "kick" Int Int
+ // --------------------------------------
+    rule <k> Flap . kick LOT BID => Vat . move MSGSENDER THIS LOT ~> KICK +Int 1  ... </k>
+         <msg-sender> MSGSENDER </msg-sender>
+         <this> THIS </this>
+         <currentTime> NOW </currentTime>
+         <flap-bids> M => M[KICK +Int 1 <- Bid(BID, LOT, MSGSENDER, 0, NOW +Int 172800)] </flap-bids>
```

Comment:
`tau` is `file`able, so we shouldn't be hardcoding the default 172800, bet using the contents of `flap-tau` cell.